common-rendering/src/components/keyEvents.tsx

```diff
@@ -0,0 +1,187 @@
+// ----- Imports ----- //
+
+import { css } from "@emotion/react";
+import type { SerializedStyles } from "@emotion/react";
+import { FC } from "react";
+import { textSans, headline } from "@guardian/src-foundations/typography";
+import { palette, remSpace } from "@guardian/src-foundations";
```

Comment:
You don't need to import the entire `palette`, you can just import the objects you need, like `sport` and `news` or `text`. We did a big mob with Simon last year going through removing all these palette imports - I think the intention is to remove this export at some point to prevent this.

Reply:
I see, thanks for spotting this, do I need to do the import directly from @guardian/src-foundations such as this:
import { remSpace, culture, news, opinion, sport } from "@guardian/src-foundations";

Reply:
Oh OK thanks for spotting 👍